lib/data-serializer.js

```diff
@@ -8,11 +8,9 @@ class DataSerializer {
     pollingRate,
     serializeSegments
   }) {
-    this._splitioApiBinding = new SplitioApiBinding({
-      splitioApiKey,
-      pollingRate,
-      serializeSegments
-    })
+    this._splitioApiBinding = new SplitioApiBinding({ splitioApiKey })
+    this.pollingRate = pollingRate || 300
```

Comment:
nit: adding the units to the variable name is helpful.  E.g., `pollingRateSeconds`